File: courses/csharp-introduction/explore_the_countries.cs
// pro

using System;

public class Program
{
    public static void Main(string[] args)
    {
        int population = Convert.ToInt32(Console.ReadLine());
        int area = Convert.ToInt32(Console.ReadLine());
        // your code goes here
        if (population < 10_000 && area < 10_000)
        {
            Console.WriteLine("small country");
        }
    }
}

File: courses/java-introduction/explore_the_countries.java
// pro

import java.util.Scanner;

class Demo {
    public static void main(String[] args) {

        Scanner read = new Scanner(System.in);
        int population = read.nextInt();
        int area = read.nextInt();
        // Complete the code
        if (population < 10_000 && area < 10_000) {
            System.out.println("small country");
        }
    }
}